packages/panels/resources/views/pages/dashboard.blade.php

```diff
@@ -1,7 +1,18 @@
 <x-filament-panels::page class="fi-dashboard-page">
+    @if (method_exists($this, 'filtersForm'))
+        <x-filament::section>
+            {{ $this->filtersForm }}
+        </x-filament::section>
+    @endif
+
     <x-filament-widgets::widgets
         :columns="$this->getColumns()"
-        :data="$this->getWidgetData()"
+        :data="
+            [
+                ...(property_exists($this, 'filters') ? ['filters' => $this->filters] : []),
```

Comment:
Isn't it cleaner to check for usage of the trait?

Reply:
@danharrin

Reply:
This is faster, less code (no import), and in reality we don't need any features of the trait, just the property to exist so we can pass it in to widgets. I don't think it hurts